xiaomi.vacuum.gen1/audio_generator/audio_de.csv

```diff
@@ -29,32 +29,32 @@ error16.wav,"Fehler 16, Benutzen Sie den Roboter auf einer flachen horizontalen
 error17.wav,"Fehler 17, Fehlfunktion der Seitenbürsten. Bitte starten Sie das System neu."
 error18.wav,"Fehler 18, Fehlfunktion des Gebläses. Bitte starten Sie das System neu."
 error19.wav,"Fehler 19, Die Dockingstation ist nicht am Strom angeschlossen."
-error_internal.wav,Ein interner Fehler ist aufgetreten. Starten Sie das System neu.
+error_internal.wav,"Ein interner Fehler ist aufgetreten. Starten Sie das System neu."
 findme.wav,"Hey, ich bin hier."
-finish.wav,Reinigung abgeschlossen
-home.wav,Fahre zurück an die Docking Station
-no_power_charging.wav,Akkustand ist gering. Benötige mehr Zeit zum Laden.
-no_power.wav,Akkustand ist gering. Fahre zurück an die Docking Station.
-no_spot_on_dock.wav,Platzieren Sie den Staubsaugerroboter im Zielbereich bevor Sie die Punktreinigung starten.
-pause.wav,Pausiert.
+finish.wav,"Reinigung abgeschlossen"
+home.wav,"Fahre zurück an die Docking Station"
+no_power_charging.wav,"Akkustand ist gering. Benötige mehr Zeit zum Laden."
+no_power.wav,"Akkustand ist gering. Fahre zurück an die Docking Station."
+no_spot_on_dock.wav,"Platzieren Sie den Staubsaugerroboter im Zielbereich bevor Sie die Punktreinigung starten."
+pause.wav,"Pausiert."
 power_off_rejected.wav,"Bewegen Sie den Roboter von der Docking Station weg, um diesen auszuschalten."
-power_off.wav,Schalte aus.
-power_resume_clean.wav,Batterie geladen. Setze Reinigung fort.
-remote_complete.wav,Verwende Fernsteuerung.
-remote.wav,Verwendung der Fernsteuerung abgeschlossen.
-restart_clean.wav,Position unbekannt. Starte neue Reinigung.
-resume_clean.wav,Setze Reinigung fort.
-resume_home.wav,Kehre zurück zur Docking Station.
-return_no.wav,Konnte nicht zu dem Startpunkt zurückkehren.
-return_yes.wav,Konnte Dockingstation nicht finden. Platzieren Sie den Roboter näher an der Dockingstation und versuchen Sie es erneut.
-spot.wav,Starte Punktreinigung.
-start.wav,Starte Reinigung.
-stop_spot.wav,Beende Punktreinigung.
-stop.wav,Beende Reinigung.
-sysupd_complete.wav,Update erfolgreich.
-sysupd_failed.wav,Firmwareupdate fehlgeschlagen. Verwende alte Version.
+power_off.wav,"Schalte aus."
+power_resume_clean.wav,"Batterie geladen. Setze Reinigung fort."
+remote_complete.wav,"Verwende Fernsteuerung."
+remote.wav,"Verwendung der Fernsteuerung abgeschlossen."
+restart_clean.wav,"Position unbekannt. Starte neue Reinigung."
+resume_clean.wav,"Setze Reinigung fort."
+resume_home.wav,"Kehre zurück zur Docking Station."
+return_no.wav,"Konnte nicht zu dem Startpunkt zurückkehren."
+return_yes.wav,"Konnte Dockingstation nicht finden. Platzieren Sie den Roboter näher an der Dockingstation und versuchen Sie es erneut.
```

Comment:
I'm not sure why you added all the quotes, but the ending quote is missing here. :)

Reply:
Oh yes, thank you! I've wanted to have all lines in the same format as some of them were already using the quotes and some don't. This lead to some strange behaviour while the loop was executed.